web/src/services/supabase.ts

```diff
@@ -1,14 +1,20 @@
 import { createClient } from "@supabase/supabase-js";
 import { LocalStorage } from "@/constants/storage";
+import invariant from "tiny-invariant";
 
-export const supabase = createClient(
-  process.env.NEXT_PUBLIC_SUPABASE_URL as string,
-  process.env.NEXT_PUBLIC_SUPABASE_ANON_KEY as string,
-);
+const { NEXT_PUBLIC_SUPABASE_URL, NEXT_PUBLIC_SUPABASE_ANON_KEY } = process.env;
```

Comment:
process.env is not an object sir 🙏 won't work

Reply:
sorted